stdlib/public/core/Print.swift

```diff
@@ -28,6 +28,8 @@ public func print(
   separator: String = " ",
   terminator: String = "\n"
 ) {
+  // Adhoc fix for 'crash at hook(output.left)' in cygwin
+  let _playgroundPrintHook : ((String)->Void)? = {_ in () }
```

Comment:
This would break playgrounds on OS X.  Please use `#if` to scope the workaround to Cygwin, file an issue in the bugtracker to implement a proper fix, and reference it here.